contracts/native/account-factory/src/commands.rs

```diff
@@ -64,23 +66,14 @@ pub fn receive_cw20(
 pub fn execute_create_account(
     deps: DepsMut,
     env: Env,
-    governance: GovernanceDetails,
+    governance: GovernanceDetails<Addr>,
     _asset: Option<Asset>,
```

Comment:
Is this asset supposed to be a default base_asset?

Reply:
This functionality will be removed (think you might have already set that task to complete, but it isn't)

Reply:
https://app.clickup.com/t/861mkce4z this one!

Reply:
re-opened the issue